### PR TITLE
🐛 winnerCount checking more understandable

### PR DESCRIPTION
**Pull request by Lebyy**
Fixed the error where it said options.winnerCount is not a positive integer.

### CODE REVIEW

File: src/Manager.js [hunk 1]
```diff
@@ -183,7 +183,7 @@ class GiveawaysManager extends EventEmitter {
             if (typeof options.prize !== 'string') {
                 return reject(`options.prize is not a string. (val=${options.prize})`);
             }
-            if (!Number.isInteger(options.winnerCount) || !options.winnerCount > 0) {
+            if (!Number.isInteger(options.winnerCount) || options.winnerCount > 0) {
```
Comment (Nico105) [inline src/Manager.js:186]:
```suggestion
            if (!Number.isInteger(options.winnerCount) || options.winnerCount < 1) {
```

File: src/Manager.js [hunk 2]
```diff
@@ -183,7 +183,7 @@ class GiveawaysManager extends EventEmitter {
             if (typeof options.prize !== 'string') {
                 return reject(`options.prize is not a string. (val=${options.prize})`);
             }
-            if (!Number.isInteger(options.winnerCount) || !options.winnerCount > 0) {
+            if (!Number.isInteger(options.winnerCount) && options.winnerCount <= 0) {
```
Comment (Nico105) [inline src/Manager.js:186]:
```suggestion
            if (!Number.isInteger(options.winnerCount) || options.winnerCount < 1) {
```

Comment (Lebyy) [reply src/Manager.js]: Oh yes Number.isInteger is present thanks!

Comment (Lebyy) [reply src/Manager.js]: I think using an `&&` statement would be better here.

Comment (Lebyy) [reply src/Manager.js]: Since if the winner count is zero it will not throw an error.

Comment (Nico105) [reply src/Manager.js]: bro we are checking if is is not a number OR if the number is below 1
your change would check "only if it is not a number AND below 1 then return"

Comment (Lebyy) [reply src/Manager.js]: Ahhhhh Alrighty thanks for correcting me xD

**Pull request merged**